draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1257,7 +1257,7 @@ Steps:
 
 # Hashing to a Finite Field {#hashtobase}
 
-The hash\_to\_base(msg) function hashes a string msg of any length into an element of a
+For a given counter ctr the hash\_to\_base(msg, ctr) function hashes a string msg of any length into an element of a
```

Comment:
I think it might be better not to introduce either parameter at this point. The sentence reads fine without them, and the ctr variable is a distraction at this point in the text.

```suggestion
The hash\_to\_base function hashes a string msg of any length into an element of a
```

Reply:
:+1: